editoast/editoast_models/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod database;
```

Comment:
```suggestion
pub mod database;

pub use database::DbConnection;
pub use database::DbConnectionPool;
```

Reply:
If you expose the `struct`  directly, maybe you don't need to expose the module anymore?

```suggestion
mod database;
```

Reply:
+1